cpp/daal/src/algorithms/kernel_function/kernel_function_rbf_dense_default_impl.i

```diff
@@ -154,6 +154,131 @@ services::Status KernelImplRBF<defaultDense, algorithmFPType, cpu>::computeInter
     return services::Status();
 }
 
+template <typename algorithmFPType, CpuType cpu>
+services::Status KernelImplRBF<defaultDense, algorithmFPType, cpu>::postGemmPart(algorithmFPType * const mklBuff,
+                                                                                 const algorithmFPType * const sqrA1i, const algorithmFPType sqrA2i,
+                                                                                 const algorithmFPType coeff, const algorithmFPType expExpThreshold,
+                                                                                 const size_t n, algorithmFPType * const dataRBlock)
+{
+    for (size_t i = 0; i < n; ++i)
+    {
+        const algorithmFPType rbf = (mklBuff[i] + sqrA2i + sqrA1i[i]) * coeff;
+        mklBuff[i]                = rbf > expExpThreshold ? rbf : expExpThreshold;
+    }
+    Math<algorithmFPType, cpu>::vExp(n, mklBuff, dataRBlock);
+    return services::Status();
+}
+
+#if defined(__INTEL_COMPILER)
+
+template <>
+services::Status KernelImplRBF<defaultDense, double, avx512>::postGemmPart(double * const mklBuff, const double * const sqrA1i, const double sqrA2i,
+                                                                           const double coeff, const double expExpThreshold, const size_t n,
+                                                                           double * const dataRBlock)
+{
+    const __m512d sqrA2iVec          = _mm512_set1_pd(sqrA2i);
+    const __m512d coeffVec           = _mm512_set1_pd(coeff);
+    const __m512d expExpThresholdVec = _mm512_set1_pd(expExpThreshold);
+
+    const size_t align = ((64 - (reinterpret_cast<size_t>(dataRBlock) & 63)) & 63) >> 3;
+
+    size_t i = 0;
+    for (; i < align; i++)
+    {
+        const double rbf = (mklBuff[i] + sqrA2i + sqrA1i[i]) * coeff;
+        mklBuff[i]       = rbf > expExpThreshold ? rbf : expExpThreshold;
+    }
+
+    for (; (i + 8) < n; i += 8)
+    {
+        const __m512d sqrDataA1Vec = _mm512_load_pd(&sqrA1i[i]);
+        __m512d sqrDataA1CoeffVec  = _mm512_mul_pd(sqrDataA1Vec, coeffVec);
+        const __m512d mklBuffVec   = _mm512_load_pd(&mklBuff[i]);
```

Comment:
sqrA1i[i] and  mklBuff[i] can be unaligned

Reply:
Fixed